scripts/templates/migrations.template.js

```diff
@@ -0,0 +1,106 @@
+const fs = require("fs")
+const { merge } = require("lodash")
+const utils = require("../../scripts/utils")
+
+const addresses = require("../../migrations/addresses")
+const settings = require("../../migrations/settings")
+const witnetRequests = require("../../migrations/witnet.requests")
+
+const WitnetPriceFeed = artifacts.require("#artifact")
```

Comment:
What's this `#artifact`?

Reply:
By now it would just get transformed into "WitnetPriceFeed", until some eventual xVM may require a specific implementation (e.g. WitnetPriceFeedOvm1)